src/App.js

```diff
@@ -110,17 +110,17 @@ const App = () => {
             <LoggingContext.Provider value={{logger}}>
                 <AppContext.Provider value={{mongoHook, debuggingTools}}>
                     <VoiceCommandsContext.Provider value={{SpeechRecognitionHook}}>
-                        <Row>
-                            <Col>
```

Comment:
Let me know if this was for a specific reason. It was causing a weird behavior in my Click Me game